tests/src/Simple/HelloWasm/HelloWasm.csproj

```diff
@@ -7,6 +7,7 @@
 	
 	<ProjectReference Include="ILHelpers.ilproj" Condition="'$(OS)' == 'Windows_NT'" />
     <IlcArg Include="-r:$(IntermediateOutputPath)\ILHelpers.dll" Condition="'$(OS)' == 'Windows_NT'" />
+    <IlcArg Include="--usesharedgenerics" />
```

Comment:
Should not be needed since shared genercis are now the default for wasm too.

Reply:
Could you remove this?

Reply:
Done.